### DISCUSSION TITLE
How to change bg color to gradient 🌈

### DISCUSSION THREAD

**Discussion by Guedesou** [anuraghazra/github-readme-stats#1742] [Q&A] [answered]
Hi, folks! How you doing? I cannot change my background color. I newbie and dont urdestand the concept to how to do it. IS this Anura said: 
**Gradient in bg_color
You can provide multiple comma-separated values in bg_color option to render a gradient, with the following format:

&bg_color=DEG,COLOR1,COLOR2,COLOR3...COLOR10** 
But, i add this after theme= selectedTheme ? dont work after all. If someone can help me. 

I wanna a repo card and stats card with gradient :(

**Comment by rickstaa** [comment, marked as answer]
@Guedesou Thanks for your question. The correct syntax for using a gradient is as follows:

```markdown
 ![99's Github Stats](https://github-readme-stats.vercel.app/api?username=Guedesou&bg_color=30,e96443,904e95&title_color=fff&text_color=fff)
````

 ![Guedesou's Github Stats](https://github-readme-stats.vercel.app/api?username=Guedesou&bg_color=30,e96443,904e95&title_color=fff&text_color=fff)

**Reply by Guedesou** [reply]
*Bro, you are not SuperMan but you saved my life!*
Thanks!

**Reply by kashifo** [reply]
Thanks, this is what i  was looking for.

**Reply by Guedesou** [reply]
Good to help, fella! Count on us!
Happy new year!
